testing/src/cypress/e2e/data/authentication-details.json

```diff
@@ -0,0 +1,7 @@
+{
+  "testEnv": {
```

Comment:
Are the tests still using this file? This info seems to be env data.

Reply:
removed the file.